src/driver/mysql/MysqlDriver.ts

```diff
@@ -788,6 +793,7 @@ export class MysqlDriver implements Driver {
      */
     protected loadDependencies(): void {
         try {
+            this.version = 1;
```

Comment:
i would rather not use `1`, `2` as version tags but rather `mysql`, `mysql2` and so on..
further, can we use this maybe as an enum for checks later on?